src/database/DataTypes/StocktakeItem.js

```diff
@@ -98,16 +98,6 @@ export class StocktakeItem extends Realm.Object {
       }
 
       let difference = quantity - this.countedTotalQuantity;
-      // In case number is entered in Actual Quantity field, but it's the same as
-      // snapshot quantity, we want to remove 'Not Counted' placeholder
-      if (difference === 0) {
-        this.batches.forEach(stocktakeBatch => {
-          stocktakeBatch.countedTotalQuantity = stocktakeBatch.snapshotTotalQuantity;
-          database.save('StocktakeBatch', stocktakeBatch);
```

Comment:
I've made a comment which you might have missed:

With this change, if there is more then one batch, and actual quantity is set to snapshot quantity we get this:
<img width="324" alt="screen shot 2018-03-19 at 8 37 18 pm" src="https://user-images.githubusercontent.com/26194949/37602975-a6ee9cf8-2bb5-11e8-876f-2a338fb43a31.png">

Reply:
Had missed, thanks!

Reply:
(changing back to countedTotalQuantity) @Chris-Petty, that was totally my bad. 
`if (difference === 0)` should ready:  `if (quantity === this.snapshotTotalQuantity)`